### DISCUSSION TITLE
Wiegand: Get the raw 7digits for the fingerprint

### DISCUSSION THREAD

**Discussion by trosenda** [arendst/Tasmota#15314] [Ideas]
Hi,
just compiled tasmota with Wiegand enabled and it is working. 
Using a combined Fingerprint/RFID/PIN terminal (Orna OR-ZS-826) I wondering about the readings for the different Methods.
The logs below indicate
a) RFID Card
b) Fingerprint
c) PIN
`12:53:38.978 MQT: tele/tasmota_D29CED/SENSOR = {"Time":"2022-04-08T12:53:38","Wiegand":{"UID”:1234567,”Size":26}}
12:53:43.399 WIF: Sending Gratuitous ARP
12:53:44.801 WIF: Checking connection...
12:53:52.685 MQT: tele/tasmota_D29CED/SENSOR = {"Time":"2022-04-08T12:53:52","Wiegand":{"UID”:12,”Size":26}}
12:54:04.767 WIF: Checking connection...
12:54:14.249 MQT: tele/tasmota_D29CED/SENSOR = {"Time":"2022-04-08T12:54:14","Wiegand":{"UID”:1234,”Size":4}}
`
The fingerprint reading seems to be the internal storage "slot" for the Fingerprint.
My Question is, how can I make it give me 7 digit number like for the RFID Card instead ?
Thanks !

**Comment by sfromis** [comment]
According to text in the [Wiegand driver](https://github.com/arendst/Tasmota/blob/development/tasmota/xsns_82_wiegand.ino), you can control something about what's displayed using `SetOption123` or `SetOption124`, and maybe something with other values available there. Can't tell if this answers your question :-)

**Reply by trosenda** [reply]
Thanks, Saw those options but they are not what I am asking for. I would like the raw 7digits for the fingerprint.
